code/modules/clothing/clothing.dm

```diff
@@ -358,7 +358,8 @@
 			to_chat(M, "<span class='warning'>[src] start[p_s()] to fall apart!</span>")
 
 //This mostly exists so subtypes can call appriopriate update icon calls on the wearer.
-/obj/item/clothing/proc/update_clothes_damaged_state()
+/obj/item/clothing/proc/update_clothes_damaged_state(damaged_state = CLOTHING_DAMAGED)
+	damaged_clothes = damaged_state
 	return
```

Comment:
not needed

Reply:
It is, this is the entire fix

Reply:
i mean the return at the bottom isn't needed. it's gonna return anyway.

Reply:
Oh lol, right.....